Sealnote/src/main/java/com/twistedplane/sealnote/NoteActivity.java

```diff
@@ -53,6 +53,7 @@
     private TagEditText mTagEditText;    /* Chips view editor for tags */
 
     private int         mBackgroundColor;
+    private boolean     mNoteChanged;
```

Comment:
See [CR0]. Initialize to false.

On other hand I think, we don't need `mNoteChanged` at first place. We already know the original state of note in object `mNote` which is either `null` for new notes and loaded via `loadNote` for saved notes. You can add a predicate function `isNoteChanged` which can compare current state with original state. This way, you get less state to manage which is a good thing.